### ISSUE TITLE
Can't install Webstorm on Ubuntu Groovy Gorilla

### ISSUE THREAD

**Issue by bk201-** [JonasGroeger/jetbrains-ppa#38]
Hi! After updating Ubuntu to 20.10, I've lost Webstorm and Rider. After trying to install them,  I got the following error:

![image](https://user-images.githubusercontent.com/6812525/97333279-74dc1300-188c-11eb-8048-110ebd6354bc.png)

Then I tried to install lib32gcc1 and got the next error:

![image](https://user-images.githubusercontent.com/6812525/97333761-f3d14b80-188c-11eb-9a51-3e50caca959d.png)

It seems like the package was renamed.

Could you please update dependencies to support Ubuntu 20.10?

**Comment by epistemex** [comment]
I ran into the same problem with Webstorm. Got around it for now by doing the steps from the preinstall and postinstall scripts, manually. I'm not sure why lib32gcc1 is needed.

https://github.com/JonasGroeger/jetbrains-ppa/tree/master/packages/webstorm

**Comment by bk201-** [comment]
@epistemex. Yep, it works!

I suggest that one of these libraries required lib32gcc1

in /build-single-deb
```bash
    --depends "lib32ncurses5 | libncurses5" \
    --depends "default-jdk | oracle-java7-installer | oracle-java8-installer" \
    --depends "libc6-i386" \
    --depends "lib32stdc++6" \
    --depends "lib32z1" \
    --depends "lib32z1-dev" \
```

**Comment by bk201-** [comment]
It seems like it was fixed in #34 but packages weren't updated. Just waiting
